template/golang-http-armhf/main.go

```diff
@@ -60,11 +61,19 @@ func makeRequestHandler() func(http.ResponseWriter, *http.Request) {
 func main() {
 	s := &http.Server{
 		Addr:           fmt.Sprintf(":%d", 8082),
-		ReadTimeout:    3 * time.Second,
-		WriteTimeout:   3 * time.Second,
+		ReadTimeout:    mustParseDuration(os.Getenv("read_timeout")),
+		WriteTimeout:   mustParseDuration(os.Getenv("write_timeout")),
 		MaxHeaderBytes: 1 << 20, // Max header of 1MB
 	}
 
 	http.HandleFunc("/", makeRequestHandler())
 	log.Fatal(s.ListenAndServe())
 }
+
+func mustParseDuration(v string) time.Duration {
+	d, err := time.ParseDuration(v)
+	if err != nil {
+		panic(err)
```

Comment:
I am not keen on the panic, I'd much rather we wrote a `log` message and used the default.

See how the watchdogs handle this please. cc @ewilde